### PR TITLE
LPS-187257

### PR DESCRIPTION
**Pull request by antonio-ortega**
Forward port of [LPS-135925](https://liferay.atlassian.net/browse/LPS-135925), to be more specific [this commit](https://github.com/liferay/liferay-portal-ee/pull/24793/commits/774599703cea52f03638c6a9f4f9fefa5cbc2860).

[Previous PR](https://github.com/liferay-frontend/liferay-portal/pull/3464).

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js [hunk 1]
```diff
@@ -32,7 +32,13 @@ function getSessionClickFormData(cmd) {
 }
 
 function getSessionClickURL() {
-	return `${Liferay.ThemeDisplay.getPortalURL()}${Liferay.ThemeDisplay.getPathMain()}/portal/session_click`;
+	let sessionClickURL = `${Liferay.ThemeDisplay.getPortalURL()}${Liferay.ThemeDisplay.getPathMain()}/portal/session_click`;
+	
+	if (Liferay.ThemeDisplay.getScopeGroupId()) {
```
Comment (dsanz) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js:37]: As per the way we generate `Liferay.ThemeDisplay` object, the call to this function would never return falsy result. This is consistent what with we're assuming in the tests, where we always expect the `doAsGroupId` parameter. I'm fine with this check though

**Comment by antonio-ortega** [comment]
ci:test:relevant

**Comment by antonio-ortega** [comment]
ci:test:stable

**Comment by antonio-ortega** [comment]
ci:forward

**Comment by antonio-ortega** [comment]
ci:forward:force

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js (hunk 1 above)
Comment (antonio-ortega) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js:37]: Actually, this seems to be an issue in final review by Brian:

> _We should only set that param if we explicitly set doAsGroupId_

Although I'm not 100% sure how to check we explicitly set that param. Would be enough to check the scopeGroupId > 0?

**Pull request closed without merging**